docs/en/tutorials/getting-started.rst

```diff
@@ -228,33 +227,196 @@ entity definition:
          * @var string
          */
         protected $name;
+    }
 
-        public function getId()
+When creating entity classes, all of the fields should be protected or private (not public).
+
+Adding the Entity behaviors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You have two options to define to define methods in your entity:
+**getters/setters**, or **mutators and DTOs**,
+respectively for **anemic models** or **rich models**.
+
+**Anemic models: Getters and setters**
+
+The most popular method is to create two kinds of methods to
+**read** (getter) and **update** (setter) the object's properties.
+
+Some fields such as ``$id`` are unlikely to be changed, so it's ok to
+omit them.
+
+.. note::
+
+    Setters are unrelated to Doctrine itself, because the ORM does not
```

Comment:
It is true that they are `unrelated`, but that's not how it should be explained to the reader.

I think a better approach is to say something like:

 > Doctrine ORM does not use any of the methods you defined: it uses reflection to read and write values to your objects, and will never call methods, not even `__construct`